### DISCUSSION TITLE
Brakeman 5.2.0 Released

### DISCUSSION THREAD

**Discussion by presidentbeef** [presidentbeef/brakeman#1661] [Announcements]
* Initial Rails 7 support ([#1653](https://github.com/presidentbeef/brakeman/issues/1653))
* Require Ruby 2.5.0+ ([#1649](https://github.com/presidentbeef/brakeman/issues/1649))
* Fix issue with calls to `foo.root` in routes ([#1640](https://github.com/presidentbeef/brakeman/issues/1640))
* Ignore `I18n.locale` in SQL queries ([#1597](https://github.com/presidentbeef/brakeman/issues/1597))
* Do not treat `sanitize_sql_like` as safe
* Add new checks for unsupported Ruby and Rails version
* Bundled version of ruby_parser updated to 3.18.1

<hr /><em>This discussion was created from the release <a href='https://github.com/presidentbeef/brakeman/releases/tag/v5.2.0'>5.2.0</a>.</em>

**Comment by heliocola** [comment]
@presidentbeef : first and foremost: :THANK_YOU: for all your time and work on :brakeman:.
I updated to 5.2.0 (from 5.0.4) and it was 99% painless! 👍
